notify/email/email.go

```diff
@@ -353,3 +353,14 @@ func (a *loginAuth) Next(fromServer []byte, more bool) ([]byte, error) {
 	}
 	return nil, nil
 }
+
+func (n *Email) getPassword() string {
+	if len(n.conf.AuthPassword) > 0 {
+		return string(n.conf.AuthPassword)
+	}
+	content, err := os.ReadFile(string(n.conf.AuthPasswordFile))
+	if err != nil {
+		return ""
```

Comment:
we should return an error, rather than silently discarding the issue.

Reply:
Good catch, fixed.  My first instinct was to have this new func treat both the "raw" and "file" fields being unset as an error, and return that error.  But the "upstream" code already has a check for that case, so I just had the new func return "" if both fields are unset.